draft-ietf-trans-gossip.md

```diff
@@ -1019,15 +1016,17 @@ are those who do not have a Trusted Auditor relationship.
 
 One of the most powerful attacks possible in the CT ecosystem is a
 trusted log that has actively decided to be malicious. It can carry
-out an attack in two ways:
+out an attack in at least two ways:
 
 In the first attack, the log can present a split view of the log for
 all time. The only way to detect this attack is to resolve each view
 of the log to the two most recent STHs and then force the log to
 present a consistency proof. (Which it cannot.) This attack can be
 detected by CT auditors participating in STH Pollination, as long as
 they are explicitly built to handle the situation of a log
-continuously presenting a split view.
+continuously presenting a split view. We highly recommend they
```

Comment:
Isn't language like "we highly recommend" more the language of a report?
Could we phrase it more in terms of MAY, SHOULD, MUST?

Reply:
Well, this document is specifying the mechanism and standards of gossiping, not auditing. There's lot of things an auditor should do, and we don't talk about 90% of them.

Reply:
OK.